server/index.js

```diff
@@ -19,6 +19,28 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  db.query('select "productId", "name", "price", "image", "shortDescription" from "products"')
+    .then(result => res.json(result.rows))
+    .catch(err => next(err));
+});
+
+app.get('/api/products/:id', (req, res, next) => {
+  if (req.params.id <= 0) res.sendStatus(400);
```

Comment:
Be sure to `return res.sendStatus(400)` or else your middleware will continue to execute and continue to query the database with a bad value (and potentially attempt to respond again, which will crash the server).